src/components/CompareResults/ResultsMain.tsx

```diff
@@ -54,7 +54,7 @@ function ResultsMain() {
     subtitle: style({
       ...FontsRaw.BodyDefault,
       fontSize: '15px',
-      lineHeight: '20px',
+      lineHeight: '1.5',
```

Comment:
If you update the line height in`h1` and `h2` in `typography.ts`, you can delete this line.